templates/web/login/common/macros.html.twig

```diff
@@ -58,7 +58,7 @@
             {% if provider.display() %}
         <li>
             <a href="{{ path('login_authentication_provider_authenticate', { 'providerId' : provider.getId() } ) }}">
-                <img src="{{ provider.getIconURI() }}" />
+                <img title="{{ provider.getTitle() }}" alt="{{ provider.getTitle() }}" src="{{ provider.getIconURI() }}" />
```

Comment:
Better to move `title` attribute to `a` tag (this parent node)

Reply:
yes